_pages/security-events.md

```diff
@@ -232,79 +227,36 @@ To configure your application to receive notifications from login.gov, supply lo
 
 Login.gov notifies for these events from the [OpenID RISC Event Types][openid-risc-events]:
 
-- [Account Purged](#account-purged)
-- [Identifier Recycled](#identifier-recycled)
-
-#### Account Purged
-
-Login.gov pushes this event when a user deletes their account.
-
-The **event_type** for this is:
-```
-https://schemas.openid.net/secevent/risc/event-type/account-purged
-```
-
-The event payload has
+{% assign outgoing_events = site.data.risc | where: 'direction', 'outgoing' %}
 
-* **subject**
-    An event should will a **subject** object, with the following keys:
+{%- for event in outgoing_events %}
+- [{{ event.friendly_name }}](#{{ event.friendly_name | slugify }})
+{%- endfor %}
 
-    * **subject_type** (string)
-      Will be **iss-sub**, this indicates the **sub** is the subject provided by the original issuer (login.gov)
+{% for event in outgoing_events %}
 
-    * **iss** (string)
-      This is login.gov's issuer, the root URL for login.gov. In the agency integration environment, this is `https://idp.int.identitysandbox.gov`
+#### {{ event.friendly_name }}
 
-    * **sub** (string)
-      The UUID identifying the user. This is the same as the  `sub` inside the `id_token` JWT in the [OpenID Token endpoint]({{site.baseurl}}/#token-response).
-
-Example:
-
-```json
-{
-  "https://schemas.openid.net/secevent/risc/event-type/account-purged": {
-    "subject": {
-      "subject_type": "iss-sub",
-      "iss": "https://idp.int.identitysandbox.gov",
-      "sub": "<$SUB>"
-    }
-  }
-}
-```
-
-#### Identifier Recycled
-
-Login.gov pushes this event when a user removes an email address from their account, freeing up the email address as an identifier.
+{{ event.description | markdownify }}
 
 The **event_type** for this is:
+
 ```
-https://schemas.openid.net/secevent/risc/event-type/identifier-recycled
+{{ event.event_type }}
 ```
 
-The event payload has
+The event payload has:
 
-* **subject**
-    An event should will a **subject** object, with the following keys:
-
-    * **subject_type** (string)
-      This will be **email**
-
-    * **email** (string)
-      This is the email address that no longer belongs to any user.
+{% include schema.html schema=event.payload_schema %}
 
 Example:
 
 ```json
-{
-  "https://schemas.openid.net/secevent/risc/event-type/identifier-recycled": {
-    "subject": {
-      "subject_type": "email",
-      "email": "<$EMAIL>"
-    }
-  }
-}
+{{ event.example_payload | jsonify | pretty_jsonify }}
 ```
 
+{% endfor %}
+
 ### Request
 
 login.gov will make a POST request to your app's `push_notification_url`, see [Configuration](#configuration) for more details on setting that up. The JWT will be signed with login.gov's private key. See the OpenID Connect guide for information on how to get login.gov's public key from the [Certificates Endpoint](/oidc/#certificates).
```

Comment:
s login.gov/Login.gov

Reply:
Great catch! Updated in d8b2384f4fd14fc6b0a4bafb2828e8a0a43d97a2